### PR TITLE
Provide more info to ScaleUpStatusProcessor

### PR DESCRIPTION
**Pull request by towca**
Add info about considered and created nodegroups to
ScaleUpStatusProcessor

### CODE REVIEW

File: cluster-autoscaler/processors/status/scale_up_status_processor.go [hunk 1]
```diff
@@ -31,6 +34,8 @@ type ScaleUpStatus struct {
 	PodsTriggeredScaleUp    []*apiv1.Pod
 	PodsRemainUnschedulable []NoScaleUpInfo
 	PodsAwaitEvaluation     []*apiv1.Pod
+	CreateNodeGroupResults  []nodegroups.CreateNodeGroupResult
+	ConsideredNodeGroups    map[string]cloudprovider.NodeGroup
```
Comment (aleksandra-malinowska) [inline cluster-autoscaler/processors/status/scale_up_status_processor.go]: nit: why not []cloudprovider.NodeGroup? Seems like it would be simpler

Comment (towca) [reply cluster-autoscaler/processors/status/scale_up_status_processor.go]: No preference here, changed

**Pull request merged**